plugin/federation/federation.go

```diff
@@ -56,9 +56,11 @@ func (f *federation) MutateConfig(cfg *config.Config) error {
 		"_Any": {
 			Model: config.StringList{"github.com/99designs/gqlgen/graphql.Map"},
 		},
-		"Entity": {
+	}
+	if len(entityFields) != 0 {
```

Comment:
Can you consider changing it as follows?
`
if len(entityFields) > 0
`

Reply:
Sure!